### ISSUE TITLE
remove OpenJ9 test specific Ant variables from test_build.xml

### ISSUE THREAD

**Issue by TianyuZuo** [eclipse/openj9#1160]
* remove OpenJ9 test specifci ant variables from build_test.xml to make tkg more generic for other projects.
* add top level test build.xml as `openj9/test/build.xml` to build openj9 test.
* all openj9 specific ant variables will be moved to top level  test build.xml to make tkg build_test.xml more generic to other projects.

**Comment by TianyuZuo** [comment]
FYI @llxia @smlambert

**Comment by llxia** [comment]
Thanks Terry. Please also move getdependency into this openj9 top level test build.xml

**Comment by TianyuZuo** [comment]
yes, getdependency will be moved too.

**Comment by TianyuZuo** [comment]
Hi @llxia  and @smlambert  

Just one more thought, do we want to move all OpenJ9 functional tests one level deep to a `functional` folder? This is going to be like 

`openj9/test`
|
|--functional
|    |--<all functional tests which previously in test folder>
|    |--openj9-test-build.xml
|--TestConfig

Advantage of this change:
* synchronize the structure of Openj9 and adoptOpenJDK
* testKitGen's build_test.xml will be generic and all iteration about BUILD_LIST will still be in TKG.
* there is no need for a top level build.xml for OpenJ9 or AdoptOpenJDK (In the contrary, if we keep the OpenJ9 tests structure, we will need a top level build.xml for OpenJ9 and AdoptOpenJDK to be called by build_test.xml)

I have two branches which reflects the changes talked above 
* keep OpenJ9 the same but with top level build.xml https://github.com/TianyuZuo/openj9/tree/buildseparate/test
* have functional folder and no top level build.xml https://github.com/TianyuZuo/openj9/tree/functional/test

**Comment by smlambert** [comment]
I believe we have arrived at another solution to how to integrate the openj9 functional tests with the rest of the test materials we use at Adopt, so this issue is no longer valid/needed.  
Please confirm @llxia @TianyuZuo

**Comment by llxia** [comment]
Yes, we can close this one. But I think the ideal of having `functional` folder may still be valid. As I am working on running functional test in AdoptOpenJDK, I noticed it is hard to run tests in both Adopt and OpenJ9 with different file structure. Tests expect same file structure to get jars/libs/files during execution. Some of tests uses $(JVM_TEST_ROOT) to locate files/jars and they expect to be the same location as TestConfig.  I did several attempts to fix the issue by updating/adding makefiles and build.xml in both Adopt and OpenJ9, I haven't be able to get a clean build so far. 
https://ci.eclipse.org/openj9/view/Test/job/Test-Sanity-Functional-JDK8-linux_ppc-64_cmprssptrs_le/

@renfeiw , any comments on this?
